### PR TITLE
Hide pager from screenshots page if not needed

### PR DESCRIPTION
**Pull request by HelenaEksler**
#111

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu PR is ready.

File: client/app/views/dashboard/screenshots/screenshots.html [hunk 1]
```diff
@@ -91,7 +91,9 @@
       max-size="maxSize"
       rotate="false"
       boundary-links="true"
-      items-per-page="itemsPerPage">
+      items-per-page="itemsPerPage"
+      ng-show="screenshots.length > itemsPerPage">
```
Comment (amitaibu) [inline client/app/views/dashboard/screenshots/screenshots.html]: The correct way should be to check if in the screenshots  RESTful results we have a `next` key

File: client/app/scripts/controllers/screenshots.js [hunk 2]
```diff
@@ -12,37 +12,28 @@ angular.module('clientApp')
...
-    $scope.screenshots = screenshots;
+    $scope.screenshots = screenshots.data;
     $scope.accessToken = Auth.getAccessToken();
 
-    $scope.filteredScreenshots = [];
     $scope.currentPage = 1;
     $scope.itemsPerPage = 50;
-
+    $scope.totalScreenshotsCount = screenshots.count;
+    $scope.pager = typeof screenshots.next !== 'undefined';
```
Comment (amitaibu) [inline client/app/scripts/controllers/screenshots.js:22]: `// Determine if a pager is needed.`

File: client/app/scripts/services/screenshots.js [hunk 3]
```diff
@@ -25,12 +25,13 @@ angular.module('clientApp')
      *
      * @returns {*}
      */
-    this.get = function(buildId) {
-      if (cache && cache[buildId]) {
-        return $q.when(cache[buildId].data);
+    this.get = function(buildId, page) {
+      page = typeof page !== 'undefined' ? page : 1;
```
Comment (amitaibu) [inline client/app/scripts/services/screenshots.js]: `page = page || 1;`

File: client/app/scripts/services/screenshots.js [hunk 4]
```diff
@@ -25,12 +25,13 @@ angular.module('clientApp')
...
+      if (cache && cache[buildId + '-' + page]) {
```
Comment (amitaibu) [inline client/app/scripts/services/screenshots.js]: `var identifier = buildId + ':' + pageNum;`

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**